homeassistant/components/withings/sensor.py

```diff
@@ -312,15 +289,18 @@ class WithingsHealthSensor(Entity):
     """Implementation of a Withings sensor."""
 
     def __init__(
-        self, data_manager: WithingsDataManager, attribute: WithingsAttribute
+        self,
+        data_manager: WithingsDataManager,
+        attribute: WithingsAttribute,
+        userid: str,
```

Comment:
Please separate words with underscore in names.
```suggestion
        user_id: str,
```

Reply:
While not proper snakecase, this variable is named this way as it is the same used by the withings api and is passed up to home assistant. I'd considered keeping it snake case but ultimately felt consistency was better than mapping it to a different name.

Reply:
Do we have to name it the same? It's still clear what it is with snakecase and we follow our standards.